configs/rhel-sst-virt-tools--subsystem.yaml

```diff
@@ -44,6 +44,45 @@ data:
       - virt-top
   package_placeholders:
     # RHEL-specific package, not in Fedora
+    - srpm_name: libguestfs-fssupport
+      build_dependencies:
+        - autoconf
+        - automake
+        - e2fsprogs-devel
+        - elfutils-libelf-devel
+        - gcc
+        - gnupg2
+        - kernel-abi-stablelists
+        - kernel-devel-uname-r
+        - kernel-rpm-macros
+        - kmod
+        - libacl-devel
+        - lzo-devel
+        - make
+        - pkgconfig(blkid)
+        - pkgconfig(libgcrypt)
+        - pkgconfig(libudev)
+        - pkgconfig(libzstd)
+        - pkgconfig(uuid)
+        - pkgconfig(zlib)
```

Comment:
Package names need to be used for placeholders.

Reply:
Why do we even need this information here?

(And if you're going to tell me it needs to be up to date, then dozens, probably hundreds of existing packages have wrong build dependencies here.)